app/lib/screens/login_screen.dart

```diff
@@ -220,18 +252,11 @@ class LoginScreenState extends State<LoginScreen> {
           child: Column(
             mainAxisAlignment: MainAxisAlignment.center,
             children: [
-              const Image(image: AssetImage('assets/MANV_transparent.png')),
+              Image(
+                  // width 80% of screen width
+                  width: MediaQuery.of(context).size.width * 0.6,
+                  image: const AssetImage('assets/MANV_transparent.png')),
               const SizedBox(height: 40),
-              Text(
-                AppLocalizations.of(context)!.loginTANHeader,
-                style: const TextStyle(
-                  fontSize: 16.0,
-                  fontWeight: FontWeight.bold,
-                ),
-              ),
-              const SizedBox(height: 4),
-              Text(AppLocalizations.of(context)!.loginTANText),
```

Comment:
And i think it would be nice to have some textual context for the login

Reply:
Done